### PR TITLE
Refactor code to modules

### PR DESCRIPTION
**Pull request by TahenyBELGUITH**
1️⃣  Created a new directory and initialized it with git.
2️⃣  Used npm init -y command to create package.json file.
3️⃣  Created the entry point for my JavaScript code called index.js
4️⃣  Created index.css file for my styles.
5️⃣  Created the main index.html file. Linked my index.js and index.css files there.
6️⃣  Created a directory called modules.
7️⃣  Re-created my Awesome books app following the rules:

> Used ES6 syntax.
> Divided my code into modules. 
> Kept all functionalities of the app without errors.
> Used let and const in a correct way.
> Displayed the current date below the navigation bar.
> For this requirement I used a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)

### CODE REVIEW

File: modules/DataBook.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      });
+    });
+}
+
+// delete function
+function deleteData(id) {
+  bookList = bookList.filter((books) => books.id !== +id);
+  localStorage.book = JSON.stringify(bookList);
+  displayBook();
+}
```
Comment (TedLivist) [inline modules/DataBook.js:39]: - [ ] In keeping with the ES6 syntax, kindly refactor your function to an arrow function.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+contactNav.addEventListener('click', () => {
+  displayLinkPage.style.display = 'none';
+  addLinkPage.style.display = 'none';
+  contactLinkPage.style.display = 'block';
+});
+
+function displayTime() {
+  document.querySelector('#calendar').innerHTML = luxon.DateTime.now().toLocaleString(luxon.DateTime.DATETIME_SHORT_WITH_SECONDS);
+  setTimeout(displayTime, 1000);
+}
```
Comment (TedLivist) [inline index.js:51]: - [ ] Kindly refactor your function to an arrow function.

File: modules/DataBook.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+// clear inputs
+export function clearData() {
+  titleBook.value = '';
+  authorBook.value = '';
+}
+
+// display books
+export function displayBook() {
+  bookSection.innerHTML = '';
```
Comment (TedLivist) [inline modules/DataBook.js:13]: - [ ] Kindly refactor your functions to be arrow functions.

  ```
  export const clearData = () => {
    ...
  }
  ```

**Review by TedLivist — COMMENTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @TahenyBELGUITH,

Good job so far!
✔️ Good documentation
✔️ Modularization of code
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @TahenyBELGUITH,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a Pull Request with failing linters

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by mikethreels — APPROVED** [review]
Hi Tahtouha, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**